signup.php

```diff
@@ -128,7 +128,13 @@
     if ($isbulksignup) {
         $error = '';
         $message = get_string('bookingcompleted', 'facetoface');
-        foreach (facetoface_get_sessions($facetoface->id) as $session) {
+        $now = time();
```

Comment:
looks like the code below is a good candidate for a new function like "facetoface_get_future_session" to reduce the code duplication.

Reply:
Done, moved into a new function in lib.php